File: MyJetPackPlugins/RealmPlugin/app/src/main/java/com/example/myjetpackapplication/plugin/realm/app/MainActivity.java
package com.example.myjetpackapplication.plugin.realm.app;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        getExternalCacheDir();
        getFilesDir();
        getCacheDir();

    }
}

File: MyJetPackPlugins/DurationPlugin/app/src/main/java/com/example/myjetpackapplication/plugin/duration/test/MainActivity.kt
package com.example.myjetpackapplication.plugin.duration.test

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import com.alibaba.android.arouter.launcher.ARouter

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun onHelloClick(view: View) {
        ARouter.getInstance().build("/api/duration").navigation()
    }
}
